src/cli/monitor.ts

```diff
@@ -59,7 +59,7 @@ export default function (program: RootCmd) {
               const uid = payload.uid;
 
               if (isMutate) {
-                const allowOrDeny = payload.res.allowed ? "✅" : "❌";
+                const allowOrDeny = payload.res.allowed ? "🔀" : "❗";
```

Comment:
```suggestion
                const allowOrDeny = payload.res.allowed ? "🔀" : "❌";
```

Reply:
Could be just me..

I'm of the opinion that the `not allowed` should be `X`. I read `!` (exclamation)  as attention, or warning. 

That said, i get why you did it, so make it more obvious what is mutate and validate.

Reply:
resolved syncro